### DISCUSSION TITLE
ESMValTool Workshop Registration Now Open!

### DISCUSSION THREAD

**Discussion by zklaus** [ESMValGroup/Community#98] [Announcements]
Dear @ESMValGroup/esmvaltool-developmentteam, everyone interested,

the next ESMValTool workshop will be held from 2023-06-27 to 2023-06-29 in a hybrid format, with the on-site part hosted at SMHI in Norrköping, Sweden.

The workshop will take three days, from Tuesday to Thursday, allowing for travel and side meetings on Monday and Friday and is open to the entire ESMValTool community, containing introductory level material and support for first developments, as well as in-depth discussions.

It also is a great opportunity to meet fellow ESMValTool developers and get to know the community!

The registration is now closed. If you would still like to participate remotely, please comment here.

The program is now finalized and available [here](https://docs.google.com/document/d/1m6WMST6V7y8QJxFeJ9q4D63jf0lcrr5pvNL02dvMnrQ/edit?usp=sharing).

We are looking forward to meeting you!

**Comment by ehogan** [comment]
For anyone wishing to share their slides for the upcoming workshop, please do so via https://github.com/ESMValGroup/Community/tree/main/Workshops/2023-06 :)

**Comment by rswamina** [comment]
@zklaus  - Are you taking suggestions for Agenda items here or elsewhere? If yes, @remi-kazeroni  and I would llike to propose a session on Observational data  in ESMValTool. Thanks!

**Comment by valeriupredoi** [comment]
hey folks, hotel-wise I am most probably going to book this one https://bookings.elite.se/ibe/index.aspx?propertyID=16759&checkIn=24/06/2023&checkOut=25/06/2023&Adults=2&child2=0&Rooms=1&LangID=6&Ratecode=619391&roomType=83220&device=desktop&utm_medium=hpa_paid&utm_content=metasearch&ChannelSite=Desktophotofinder&ChannelCode=GO&subchan=DerbySoft-Google&sitetype=mapresults&utm_source=google&utm_campaign=cpc&dsclid=53383271479529472

**Comment by bouweandela** [comment]
@zklaus I would like to propose a few topics for the agenda, is this discussion the right place?

Topics:
- Improving the configuration system
- Improving computational performance
- @ESMValGroup/technical-lead-development-team retrospective

**Reply by hb326** [reply]
@bouweandela: I just checked with @zklaus and he nodded. 
Yes, this is the place to propose topics for the agenda.
Might also be of interest to other people from @ESMValGroup/esmvaltool-developmentteam :)

**Comment by zklaus** [comment]
Thanks for the suggestions, @ESMValGroup/esmvaltool-developmentteam!

The preliminary program is now available [here](https://docs.google.com/document/d/1m6WMST6V7y8QJxFeJ9q4D63jf0lcrr5pvNL02dvMnrQ/edit?usp=sharing).

Please have a look, feel free to comment and *volunteer* for rapporteur and chat monitor slots.

**Reply by mo-tgeddes** [reply]
Hi, would it be possible for a 15 minute slot somewhere for @Jon-Lillis and I to talk about the UK Met Office updates? This would be about the Recipe Test Workflow(RTW) and the Climate Model Evaluation Workflow(CMEW).

**Reply by zklaus** [reply]
For sure! Would any of the three days work for you?

**Reply by mo-tgeddes** [reply]
Any of them are good thanks :).

**Comment by axel-lauer** [comment]
For the coding sessions, I would like to propose a session on updating the content of our website.

**Comment by rswamina** [comment]
@ESMValGroup/userengagementteam - can we plan tutorial updates for the coding sessions?

**Comment by egalytska** [comment]
Did I get it right that there is no solid plan for Monday? I would love to attend the tutorial, but still not sure about the final arrival time to Norrköping (coming by 🚊).

**Reply by zklaus** [reply]
The tutorial participants have received the information by email, albeit after @egalytska's comment here. Thanks for the participation!

**Comment by hb326** [comment]
@zklaus: Could you provide links for the Pexip/Zoom meetings? I know of a few people who are waiting for these so that they can join our meeting online. Thanks!

**Reply by glpotter** [reply]
@zklaus Is there a link for the Zoom meeting? I haven't seen one yet.

**Reply by glpotter** [reply]
@zklaus The registration form has expired. I would like to attend remotely, is it necessary to register?

**Reply by zklaus** [reply]
Hi @glpotter, I had to close the registration for logistical reasons, but I have added you manually and the link will be available at the bottom of the program shortly.

**Comment by tepmo42** [comment]
I also do not think I have had a link for the workshop yet ? I have registered for remote participation.

**Reply by zklaus** [reply]
The links will be available in the program shortly.

**Comment by zklaus** [comment]
[The program](https://docs.google.com/document/d/1m6WMST6V7y8QJxFeJ9q4D63jf0lcrr5pvNL02dvMnrQ/edit?usp=sharing) is now finalized and contains the link for remote participation.
